### ISSUE TITLE
ValueError: dictionary update sequence element #0 has length 7; 2 is required

### ISSUE THREAD

**Issue by ghost** [InstaPy/InstaPy#2287]
## Expected Behavior
Updated to most recent master and i keep getting this error now for unfollowing active users
## InstaPy configuration
```
INFO [2018-06-20 21:57:39] [eatinghalalfood]  Session ended - 2018-06-20 21:57:39
Traceback (most recent call last):
  File "/Users/piratze/Documents/GitHub/InstaPy/quickstart.py", line 50, in <module>
    session.set_dont_unfollow_active_users(enabled=True, posts=4)
  File "/Users/piratze/Documents/GitHub/InstaPy/instapy/instapy.py", line 2533, in set_dont_unfollow_active_users
    self.dont_include.update(active_users)
ValueError: dictionary update sequence element #0 has length 7; 2 is required
```

**Comment by sionking** [comment]
ohh need to fix soon

**Comment by sionking** [comment]
try to change this
under _def set_dont_include(self, friends=None):_
self.dont_include = set(friends) or {}
to:
self.dont_include = set(friends) or set()

If no friend exist it might consider as dict.

please let us know.

**Comment by ghost** [comment]
still happens with above change - i believe https://github.com/timgrossmann/InstaPy/pull/2294/files is the full fix?

**Comment by sionking** [comment]
Can you try ? I hope it just a python treat set as dict since inited with {}

**Comment by axlright** [comment]
I had this same issue and the fix at #2294 worked cleared up the issue for me.

**Comment by ghost** [comment]
yes #2294 fixed it. Thanks @sionking !
